simple.js

```diff
@@ -1381,19 +1378,22 @@ define(function(require, exports, module) {
                 // set Python's version
                 settings.set("project/python/@version", "python3");
 
-                // set pylint's flags
-                settings.set(
-                    "project/python/@pylintFlags",
-                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQLAlchemy --load-plugins pylint50"
-                );
-
                 // set PYTHONPATH
                 settings.set(
                     "project/python/@path",
                     "/home/ubuntu/.local/lib/python3.4/site-packages" +
                     ":/usr/lib/python3/dist-packages" +
                     ":/usr/local/lib/python3.4/dist-packages"
                 );
+
+                // set pylint's flags
+                settings.set(
+                    "project/python/@pylintFlags",
+                    "-d all -e E -e F --generated-members=app.jinja_env.* --ignored-classes=SQL,SQLAlchemy --load-plugins pylint50"
+                );
+
+                // update settings version
+                settings.set("project/cs50/simple/@ver", SETTINGS_VER);
```

Comment:
This moved the revision number for our plugin's settings to project settings instead of user settings. This way if project settings were reset somehow (which was probably what caused the problem), this ensures our settings would get rewritten (because revision number would be absent from default project settings in case they were reset).

This doesn't happen now because revision number is still present in user settings, so the plugin decides not to rewrite the settings.